packages/tools/devtools/devtools-core/package.json

```diff
@@ -40,8 +82,10 @@
 		"rebuild": "npm run clean && npm run build",
 		"test": "npm run test:mocha",
 		"test:coverage": "c8 npm test",
-		"test:mocha": "mocha \"dist/**/*.test.js\" --exit",
-		"tsc": "tsc",
+		"test:mocha": "npm run test:mocha:esm && npm run test:mocha:cjs",
+		"test:mocha:cjs": "mocha --recursive \"dist/test/**/*.spec.*js\" --exit -r node_modules/@fluid-internal/mocha-test-setup",
+		"test:mocha:esm": "mocha --recursive \"lib/test/**/*.spec.*js\" --exit -r node_modules/@fluid-internal/mocha-test-setup",
```

Comment:
Just for reference, I think the fact that this package's [`require`s mocha-test-setup's base file](https://github.com/microsoft/FluidFramework/blob/7621baec8ef1ca0436d3429e5714317a281a40a7/packages/tools/devtools/devtools-core/.mocharc.cjs#L8), automatically [adds that package to the dynamic `require`s](https://github.com/microsoft/FluidFramework/blob/7621baec8ef1ca0436d3429e5714317a281a40a7/packages/test/mocha-test-setup/mocharc-common.js#L14-L20). Nothing in particular against being explicit, though.

Reply:
Fixed (and wrote a script to patch my other open PRs.)

Reply:
@DLehenbauer, does your script check that mocha config has a require of that package? I know that some packages' mocha lines had the -r before we were updating. I don't know if any of them needed the -r (because they didn't require the config and get side-effect).

Reply:
It does now.